### PR TITLE
Moving forward

### PR DESCRIPTION
**Pull request by alexdowad**

### CODE REVIEW

**Comment by alexdowad** [comment]
Points raised by @bbatsov have been addressed. Should we also rename `SingleSpaceBeforeFirstArg` to `SpaceBeforeFirstArg`?

The last commit for `Style/FileName` is not complete yet.

**Comment by bbatsov** [comment]
> Should we also rename SingleSpaceBeforeFirstArg to SpaceBeforeFirstArg

Yes.

**Comment by alexdowad** [comment]
OK, @bbatsov, see what you think now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,12 @@
...
+* [#2511](https://github.com/bbatsov/rubocop/issues/2511): `--no-offense-counts` CLI option suppresses the inclusion of offense count lines in auto-generated config. ([@alexdowad][])
+* [#2504](https://github.com/bbatsov/rubocop/issues/2504): New `AllowForAlignment` config parameter for `Style/SingleSpaceBeforeFirstArg` allows the insertion of extra spaces before the first argument if it aligns it with something on the preceding or following line. ([@alexdowad][])
+* [#2478](https://github.com/bbatsov/rubocop/issues/2478): `Style/ExtraSpacing` has new `ForceEqualSignAlignment` config parameter which forces = signs on consecutive lines to be aligned, and it can auto-correct. ([@alexdowad][])
+* `Lint/BlockAlignment` aligns block end with unary operators like ~, -, or ! when such operators are applied to the method call taking the block. ([@alexdowad][])
+* [#1460](https://github.com/bbatsov/rubocop/issues/1460): `Style/Alias` supports both `prefer_alias` and `prefer_alias_method` styles. ([@alexdowad][])
```
Comment (bf4) [inline CHANGELOG.md]: @alexdowad Thanks for taking care of this! It was a little behind on my todo list :)

Comment (alexdowad) [inline CHANGELOG.md]: You're welcome. Please pull the latest code and try it out.

**Pull request merged**